tests/netstats_l2/tests/01-run.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2017 Alexandre Abadie <alexandre.abadie@inria.fr>
+#               2017 Martine Lenders <m.lenders@fu-berlin.de>
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
+import sys
+
+
+def testfunc(child):
+    child.sendline('ifconfig')
+    child.expect(r'ifconfig')
+    child.expect(r'Iface  \d+   HWaddr: ([0-9a-f]{2}:){1,}([0-9a-f]{2})')
+    child.expect(r'       Source address length: \d+')
```

Comment:
Why are you still checking the hardware address and source address length? They have *nothing* to do with the netstats.

Reply:
removed